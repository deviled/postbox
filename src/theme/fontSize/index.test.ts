import { fontSize } from '.';

describe('fontSize', () => {
  it('should match snapshot', () => {
    expect(fontSize).toMatchSnapshot();
  });
});
