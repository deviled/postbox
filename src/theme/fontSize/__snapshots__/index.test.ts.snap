// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fontSize should match snapshot 1`] = `
Object {
  "12": "0.75rem",
  "14": "0.875rem",
  "16": "1rem",
}
`;
